packages/components/src/timeline/style.scss

```diff
@@ -0,0 +1,114 @@
+
+.woocommerce-timeline {
+	ul {
+		margin: 0;
+		padding-left: 0;
+		list-style-type: none;
+
+		li {
+			margin-bottom: 0;
+		}
+	}
+
+	.woocommerce-timeline-group {
+		.woocommerce-timeline-group__title {
+			color: $studio-gray-90;
+			font-size: 12px;
+			font-weight: 600;
+			text-transform: uppercase;
+
+			margin: 0 0 $gap 0;
+		}
+
+		hr {
+			float: right;
+			width: calc(100% - #{ $gap-largest });
+
+			margin-bottom: $gap;
+
+			// Color is according to design, we should probably find a suitable color variable.
+			border: 0.5px solid #e3dfe2;
+		}
+	}
+
+	.woocommerce-timeline-item {
+		.woocommerce-timeline-item__top-border {
+			min-height: 16px;
+			border-left: 1px solid $studio-gray-10;
+			margin: 0 $gap-small;
+		}
+
+		.woocommerce-timeline-item__title {
+			display: flex;
+			align-items: center;
+			flex-direction: row;
+			justify-content: space-between;
+
+			p {
+				color: $studio-gray-80;
+				font-size: 16px;
+			}
+		}
+
+		.woocommerce-timeline-item__headline {
+			display: flex;
+			align-items: center;
+			flex-direction: row;
+			margin: $gap-smaller 0;
+
+			span {
+				padding: 0 $gap;
+			}
+
+			svg {
+				fill: $studio-white;
+				padding: $gap-smallest;
+				background: $studio-gray-10;
+				border-radius: 9999px;
+				box-sizing: content-box;
+				// We hard code the size to maintain consistent styling and spacing.
+				width: 16px;
+				height: 16px;
+
+				&.is-success {
+					background: $valid-green;
+				}
+
+				&.is-warning {
+					background: $notice-yellow;
+				}
+
+				&.is-error {
+					background: $error-red;
+				}
```

Comment:
See https://github.com/Automattic/woocommerce-payments/pull/344#discussion_r413108027 for discussions in WCPay on whether styling icons like this here is appropriate or not. Feel free to add additional comments below either here or in WCPay.